server/sonar-db-migration/src/main/java/org/sonar/server/platform/db/migration/version/v71/DbVersion71.java

```diff
@@ -36,6 +36,7 @@ public void addSteps(MigrationStepRegistry registry) {
       .add(2006, "Clean orphans in Compute Engine child tables", CleanCeChildTablesOrphans.class)
       .add(2007, "Update PERMISSION_TEMPLATES.KEYS ", UpdatePermissionTooLongTemplateKeys.class)
       .add(2008, "Increase branch type size in PROJECT_BRANCHES", IncreaseBranchTypeSizeForPullRequest.class)
+      .add(2009, "Add key type column in PROJECT_BRANCHES", AddKeyTypeInProjectBranches.class);
     ;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this empty statement. ([squid:EmptyStatementUsageCheck](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fjanos%2FSONAR-10365%2Fadd-new-columns-in-project&issues=AWGUl_50QeDOAQ7I7loL&open=AWGUl_50QeDOAQ7I7loL)

[ISSUE_KEY]: AWGUl_50QeDOAQ7I7loL